inst/assets/BS4/pkgdown.css

```
@@ -360,3 +361,27 @@ table {
 .fab {
     font-family: "Font Awesome 5 Brands" !important;
 }
+
+/* logo before TOC */
+#logo-toc {
+  display: grid;
+}
+
+/* logo homepage */
+#logo-homepage {
+  float: right;
+  width: 200px;
```

```suggestion
  width: 180px;
```

I think this looks a little nicer